xcode/Subconscious/Shared/Components/Common/Profile/UserProfileView.swift

```diff
@@ -405,7 +405,9 @@ private struct EditProfileSheetModifier: ViewModifier {
                 }
             }
             .onReceive(store.actions) { action in
-                switch(action) {
+                switch action {
+                case .refresh:
+                    app.send(.syncAll)
```

Comment:
Ensure everything is refreshed if a user is trying to manually refresh their following list.

Reply:
Another way we do this is through the slightly more compact/fancy approach of:

```swift
.onReceive(
    store.actions.compactMap(AppAction.from),
    perform: app.send
)
```

where `AppAction.from` is an extension that creates an app action from a local action, or else nil. The advantage being that the `AppAction.from` extension can be tested.

Non-blocking.

Reply:
Clean! I like that this moves the mapping nearer to the model and away from the view.